src/languages/es.js

```diff
@@ -150,7 +151,9 @@ export default {
         deleteConfirmation: '¿Estás seguro de que quieres eliminar este comentario?',
     },
     reportActionsView: {
-        beFirstPersonToComment: 'Sé el primero en comentar',
+        beginningOfChatHistory: 'Aquí comienza tu historial de conversaciones con',
+        beginningOfChatHistoryPrivate: 'This is the beginning of the private',
```

Comment:
Please ping again the channel.

Reply:
@parasharrajat i combined beginningOfChatHistory and beginningOfChatHistoryPrivate as single sentence.
beginingOfChatHistroyPrivate: ({name}) => `This is the beginning of the private ${name} room, invite others by @mentioning them.`,
as suggested on issue